tests/acceptance/href-to-test.js

```diff
@@ -12,51 +13,66 @@ function assertAnchorIsActive(selector, assert) {
 module('Acceptance | href to', function(hooks) {
   setupApplicationTest(hooks);
 
+  test('href renders and updates based on router.currentState', async function (assert) {
+    await visit('/about');
+
+    assert.equal($('#link-to-links a:contains(About)').attr('href'), '/about')
+
+    const aboutController = this.owner.lookup('controller:about');
+
+    run(function() {
+      aboutController.set('section', 'two');
+    });
+
+    await settled();
+    assert.equal($('#link-to-links a:contains(About)').attr('href'), '/about?section=two')
```

Comment:
perhaps it would be useful to also test the behaviour of `<LinkTo>` alongside `{{href-to}}` to ensure that they have the same behaviour?

Reply:
Yeah I agree and will add once I iterate post Ember update.

Reply:
Fixed and updated